apps/prairielearn/elements/pl-file-upload/pl-file-upload.mustache

```diff
@@ -31,3 +47,4 @@ $(() => {
     </div>
   </div>
 </div>
+{{/question}}
```

Comment:
Fixed.

Reply:
It's still giving the notification about missing a blank line?

You should enable automatically adding a blank line at the end of every file on save in your editor, otherwise you'll see this error all over the place in files you commit.